nixos/modules/services/networking/gnunet.nix

```
@@ -148,17 +149,19 @@ in
     # so install them globally.
     environment.systemPackages = [ cfg.package ];
 
+    environment.etc."gnunet.conf".text = configFile;
+
     systemd.services.gnunet = {
       description = "GNUnet";
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
       path = [ cfg.package pkgs.miniupnpc ];
```

```suggestion
      wantedBy = [ "multi-user.target" ];
      restartTriggers = [ configFile ];
      path = [ cfg.package pkgs.miniupnpc ];
```

The code in this PR is inadvertently tricky. Changes to your configuration will not impact any running services anymore. Just pop in a `restartTriggers` and you're good to go!